src/integ_test_resources/ios/sdk/integration/cdk/custom_resources/iot_custom_authorizerUserPass_function/iot_custom_authorizerUserPass.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+
+
+def handler(event, __):
+    print(f"### handler: {event}")
+    token = event["token"].lower()
+    effect = "Allow" if token == "allow" and event["protocolData"]["mqtt"]["username"].startswith("aws-sdk-user?SDK=iOS&Version=") else "Deny"
```

Comment:
To break up the long line and get some semantic verbiage going in the last line,
```suggestion
    mqtt_user = event["protocolData"]["mqtt"]["username"]
    ios_token = "aws-sdk-user?SDK=iOS&Version="
    effect = "Allow" if token == "allow" and mqtt_user.startswith(ios_token) else "Deny"
```